app/src/main/res/layout/item_metadata_sample_image_with_mask.xml

```diff
@@ -0,0 +1,47 @@
+<?xml version="1.0" encoding="utf-8"?>
+
+<androidx.constraintlayout.widget.ConstraintLayout xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:app="http://schemas.android.com/apk/res-auto"
+    xmlns:tools="http://schemas.android.com/tools"
+    android:layout_marginStart="3dp"
+    android:layout_height="wrap_content"
+    android:layout_width="wrap_content"
+    android:clipChildren="false"
+    android:clipToPadding="false"
+    android:paddingTop="6dp"
+    android:paddingEnd="5dp"
+    android:paddingStart="0dp">
+
+    <androidx.constraintlayout.widget.ConstraintLayout
```

Comment:
Also, there's no need for needing a `ConstraintLayout` inside another `ConstraintLayout`. You can achieve whatever needed with only one.